Update container-service-deploy-spring-boot-app-on-kubernetes.md

lastcoolnameleft opened the pull request:
dockerDirectory should be where the Dockerfile is; however, it's pointing to src/main/docker, but that directory doesn't exist in that repo.  It's in the root directory, so changed to "."

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

lastcoolnameleft committed:
Update container-service-deploy-spring-boot-app-on-kubernetes.md

dockerDirectory should be where the Dockerfile is; however, it's pointing to src/main/docker, but that directory doesn't exist in that repo.  It's in the root directory, so changed to "."
